pkg/certificate/ca.go

```diff
@@ -21,12 +21,13 @@ import (
 
 	kubeoneapi "github.com/kubermatic/kubeone/pkg/apis/kubeone"
 	"github.com/kubermatic/kubeone/pkg/ssh"
-	"github.com/kubermatic/kubeone/pkg/util"
+	"github.com/kubermatic/kubeone/pkg/util/context"
+	"github.com/kubermatic/kubeone/pkg/util/runner"
```

Comment:
I suggest to move `runner` out of `util`.

Reply:
I've thought about that as well, but I wasn't sure. `runner` is something you're not going to use standalone—you're going to use it in some package to run something, so it may make sense to be a utility (`util`) package.

Reply:
Done.